csv_optional_support.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2020 Swift Navigation Inc.
+ * Contact: Swift Navigation <dev@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#ifndef CSV_OPTIONAL_SUPPORT_H
+#define CSV_OPTIONAL_SUPPORT_H
+
+namespace io {
+
+namespace detail {
+
+template <class overflow_policy, class T>
+bool parse(char *col, optional<T> &x) {
+  T x_value;
+  if (parse(col, x_value)) {
```

Comment:
Won't you need to use the `overflow_policy` here?
```suggestion
  if (parse<overflow_policy>(col, x_value)) {
```